sympy/parsing/sympy_parser.py

```diff
@@ -979,9 +979,10 @@ def parse_expr(s, local_dict=None, transformations=standard_transformations,
 
     stringify_expr, eval_expr, standard_transformations,
     implicit_multiplication_application
-
     """
-
+    if type(transformations) is str:
```

Comment:
I think it makes sense to check that transformations really is `"all"` and not just a string.

Also, please update the documentation pointing out this option and add a test. The example you showed in the post works well as a test.